web-marketplace/src/components/templates/ProjectDetails/ProjectDetails.tsx

```diff
@@ -165,6 +166,13 @@ function ProjectDetails(): JSX.Element {
     }),
   );
 
+  const slug = offchainProjectByIdData?.data?.projectById?.slug;
```

Comment:
please be aware that this will only redirect from **off chain** based UUID 
I was wondering if that made sense to do that for on chain based ids as well, happy to add this case as well if you think that makes sens @clevinson @aaronc

Reply:
The slug should probably be preferable in all cases

Reply:
ok this is the case now too
I've also created this sanity PR https://github.com/regen-network/regen-sanity/pull/57 to update the description